src/components/AttachmentModal.js

```diff
@@ -318,12 +322,30 @@ function AttachmentModal(props) {
             >
                 {props.isSmallScreenWidth && <HeaderGap />}
                 <HeaderWithBackButton
-                    title={props.headerTitle || translate('common.attachment')}
+                    title={props.headerTitle || translate(isAttachmentReceipt ? 'common.receipt' : 'common.attachment')}
                     shouldShowBorderBottom
                     shouldShowDownloadButton={props.allowDownload}
                     onDownloadButtonPress={() => downloadAttachment(source)}
                     shouldShowCloseButton={!props.isSmallScreenWidth}
                     shouldShowBackButton={props.isSmallScreenWidth}
+                    shouldShowThreeDotsButton={isAttachmentReceipt}
+                    threeDotsMenuItems={[
+                        {
+                            icon: Expensicons.Camera,
+                            text: props.translate('common.replace'),
+                            onSelected: () => {}, // TODO
+                        },
```

Comment:
NAB we have a specific [issue](https://github.com/Expensify/Expensify/issues/295248) for adding the replace functionality. Maybe we should leave adding this stuff for that issue.